wandb/sklearn/__init__.py

```diff
@@ -0,0 +1,1215 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import wandb
+import time
+import itertools
+import sklearn
+import numpy as np
+import scipy as sp
+from wandb.sklearn.utils import *
+from sklearn.base import clone
+from joblib import Parallel, delayed
+from sklearn import model_selection
+from sklearn import datasets
+from sklearn import metrics
+from sklearn.metrics import roc_curve, auc, precision_recall_curve, average_precision_score
+from sklearn.metrics import (brier_score_loss, precision_score, recall_score, f1_score)
+from sklearn.metrics import silhouette_score, silhouette_samples
+from sklearn.preprocessing import label_binarize
+from sklearn.preprocessing import LabelEncoder
+from sklearn.calibration import calibration_curve
+from sklearn import naive_bayes
+from sklearn.utils.multiclass import unique_labels, type_of_target
+from sklearn.calibration import CalibratedClassifierCV, calibration_curve
+from sklearn.linear_model import LogisticRegression
+from warnings import simplefilter
+# ignore all future warnings
+simplefilter(action='ignore', category=FutureWarning)
+
+def round_3(n):
+    return round(n, 3)
+def round_2(n):
+    return round(n, 2)
+chart_limit = 1000
+def get_named_labels(labels, numeric_labels):
+        return np.array([labels[num_label] for num_label in numeric_labels])
+
+"""
+Generates all sklearn classifier plots supported by W&B.
```

Comment:
I thought function docstrings should be inside the i.e. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tf2.py#L32

I’m now curious if python parses this correctly, regardless we should probably configure black or a python formatted to enforce a style.

Reply:
moved [here](https://github.com/wandb/client/pull/719/commits/fdd28b9e5049bf95f6a4aab00ef316066bf9df84) :)